src/routes/(auth)/+layout.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+	import './layout.css';
+	import '../../app.css'
+	import { page } from '$app/stores'
+	import ucabLogo from '$lib/images/logo-ucab.webp'
+	import firstLoginImage from '$lib/images/edificio.png'
```

Comment:
```suggestion
	import loginImage from '$lib/images/login-image.png'
```
Hacer eso si se decide cambiarle el nombre a login-image.png, sino:
```suggestion
	import edificio from '$lib/images/edificio.png'
```